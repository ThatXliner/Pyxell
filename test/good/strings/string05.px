
n = 0
s = "test" * 3*10^^6

for i in 0...3*10^^6 do
    s' = s
    if s'[4*i] == s[-4*i-1] do
        n += s'.length

print n
