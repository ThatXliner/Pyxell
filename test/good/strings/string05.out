36000000000000
